strings/src/main/res/values/strings.xml

```diff
@@ -278,6 +278,8 @@
     <!-- Text next to checkbox to allow the user to toggle audio recording on and off -->
     <string name="record_audio">Record audio</string>
 
+    <string name="background_audio_permission_explanation">This form records audio in the background. You need to grant Collect permission to use your microphone to fill it in.</string>
```

Comment:
Second sentence feels a bit hard to parse, I think because of "Collect." Maybe "You must grant permission to use your microphone. If you do not, you will not be able to open this form."